Dockerfile

```diff
@@ -14,7 +14,7 @@ COPY db db
 RUN cargo install --path .
 
 
-FROM debian:bullseye-slim
-RUN apt-get update && apt-get install -y libssl1.1 && rm -rf /var/lib/apt/lists/*
+FROM debian:bookworm-slim
+RUN apt-get update && apt-get install -y openssl && rm -rf /var/lib/apt/lists/*
```

Comment:
Oh nice, might this help with the ssl build failures I was struggling with yesterday? The commits to master were trying to fix it, I ended up staying on an older rust compiler/clap version to work around it. I'll retry newer versions after merging this, hopefully it helps.

Reply:
Yeah, I was getting ssl errors on startup and this is what fixed it for me.